packages/runtime/container-runtime/src/test/garbageCollection.spec.ts

```diff
@@ -230,6 +231,44 @@ describe("Garbage Collection Tests", () => {
 				assert(gc.gcEnabled, "gcEnabled incorrect");
 				assert.equal(gc.latestSummaryGCVersion, 1, "latestSummaryGCVersion incorrect");
 			});
+			describe("gcEnforcementMinCreateContainerRuntimeVersion overriding sweepEnabled true for some existing containers", () => {
```

Comment:
https://github.com/microsoft/FluidFramework/pull/13985/files/9fd0e174f7105671eacb0f1fe86efd5cee9bc283#r1096322059

Highlighting Navin's comment about the name being wordy - I like it but open to other suggestions :)